server.py

```diff
@@ -0,0 +1,100 @@
+"""
+Copyright Schrodinger, LLC. All rights reserved.
+"""
+
+import csv
+import os
+import requests
+import sys
+import tempfile
+from flask import Flask
+from flask import jsonify, request, render_template
+from flask_cors import CORS, cross_origin
+from  rdkit import Chem
+from  rdkit.Chem import rdDepictor
+from functools import partial
+
+
+app = Flask(__name__)
+CORS(app)
+
+sys.path.insert(0, "/software/lib/Darwin-x86_64/rdkit-Release_2018_03_2/lib/python3.6/site-packages/")
+
+
+def runOnMolecule(function):
+    json = request.json
+    struct = json['struct']
+    mol = Chem.MolFromMolBlock(struct)
+    function(mol)
+    struct = Chem.MolToMolBlock(mol, kekulize=False)
+    json['struct']=struct
+    return jsonify(json)
+
+
+def cleanUp(molecule):
+    rdDepictor.SetPreferCoordGen(True)
+    rdDepictor.Compute2DCoords(molecule)
+
+
+
+@app.route('/info', methods=['GET'])
+def info():
+   info = {'imago_version':'no_imago'}
+   return jsonify(info)
+
+
+
```

Comment:
nit there are some cases of extra lines between functions.  I'll just comment here and avoid pointing them all out as this is still an early POC.